### ISSUE TITLE
Disconnected reconnected messages when dying.

### ISSUE THREAD

**Issue by Rysanlos** [rfalias/project_zomboid_bot#13]
The deaths message are working great with the new update, but it's still doing disconnected reconnected messages.
I tried this:
```
                if "removed" in l:
                    print("at dc")
                    print(l)
                    user = ls[3].strip('"')
                    await channel.send(f"{user} has disconnected!")
                    break
                if "allowed to join" in l:
                    print("Connected")
                    print(l)
                    user = ls[3].strip('"')
                    await channel.send(f"{user} has joined!")
                    break
                if "died at (" in l:
                    print("At Died")
                    print(l)

                    user = ls[3].strip('"')
                    await channel.send(f"{user} has died!")
                    break
                break
```
Join is working, dying too without disconnected/reconnected, but I can't make disconnecting works.

I tried "removed", "removed connection" and "removed connection index" but nothing works.
Thanks

_Originally posted by @Rysanlos in https://github.com/rfalias/project_zomboid_bot/issues/8#issuecomment-1025317430_

**Comment by rfalias** [comment]
The game actually disconnects and reconnects players when they die, so that's why. I can look at suppressing then after a death

**Comment by rfalias** [comment]
Fixed in #14 
Handles deaths by ignoring the next two immediate Connect/Disconnect events. Resets the state on full removed index. 

Also the reason removed connection index didn't work is the reverse read only reads the last line and then stops. That line is sent first, then the 'disconnect'. I skipped one iteration when I see disconnect to verify the index was removed and suppressed extra noise this way.
